app/views/campaigns/_campaigns.html.haml

```diff
@@ -47,34 +47,7 @@
     %tbody.list
       - presenter.campaigns_by_num_courses.each do |campaign|
         - campaign_presenter = CoursesPresenter.new(current_user, campaign.slug)
-        %tr{"data-link" => "#{campaign_path(campaign.slug)}"}
-          %td.title{role: "button", tabindex: "0"}
-            = campaign.title
-            %span.creation-date.hidden
-              = campaign.created_at
-          %td
-            %span.num-courses-human
-              = number_to_human(campaign_presenter.courses.count)
-            %span.num-courses.hidden
-              = campaign_presenter.courses.count
-          %td
-            %span.articles-created-human
-              = number_to_human(campaign_presenter.courses.sum(:new_article_count))
-            %span.articles-created.hidden
-              = campaign_presenter.courses.sum(:new_article_count)
-          %td
-            %span.characters-human
-              = number_to_human(campaign_presenter.word_count)
-            %span.characters.hidden
-              = campaign_presenter.word_count
-          %td
-            %span.views-human
-              = number_to_human(campaign_presenter.courses.sum(:view_sum))
-            %span.views.hidden
-              = campaign_presenter.courses.sum(:view_sum)
-          %td
-            %span.students
-              = campaign_presenter.campaign.students_without_nonstudents.count
+        = render 'campaigns/row', presenter: campaign_presenter, campaign: campaign
```

Comment:
It is also used in [dashboard/index.html.haml](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/1110/files#diff-6ba95d8ef933598e22ac06d1e009b57eR135). The convention of having `_row.html.haml` just be the `<tr>` markup itself is modeled after the [courses/row](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/views/courses/_row.html.haml) partial